src/cosmos/CosmosEngine.ts

```diff
@@ -203,6 +222,7 @@ export class CosmosEngine extends CurrencyEngine<
         progress
       this.updateOnAddressesChecked()
     }
+    this.otherData.archivedTxLastCheckTime = Date.now()
```

Comment:
Just a comment for future consideration:

`CurrencyEngine.walletLocalData` already has (but poorly named) `lastTransactionQueryHeight`. To avoid having custom values in `otherData`, this could be used. I would suggest renaming it, but it's already saved on disk for EVM users.